### ISSUE TITLE
Pseudo-code for Flow RWR

### ISSUE THREAD

**Issue by baoxiaoyi** [KwaiVGI/VideoAlign#7]
Great work! For the Flow RWR, can you provide some pytorch-style pseudo-code for better understanding? 
By the way, in the training process with FlowRWR, do you limit the noise timestep? Since high noise may suppress the reward score.

**Comment by yifan123** [comment]
Thanks for your interest in our work. Flow-RWR is a simple reward weighted regression, which is essentially standard flow matching training but uses the reward as the weight for the loss function.

**Comment by SHYuanBest** [comment]
@yifan123 Hi! Thanks for great work! Do you use the [raw reward score](https://github.com/KlingTeam/VideoAlign/blob/main/inference.py#L226) directly as the weight coefficient, or do you apply torch.exp or some other normalization? Just wondering since the raw scores look like they're around 0~5 on average.
